conans/client/detect.py

```diff
@@ -189,7 +189,7 @@ def detect_defaults_settings(output):
     """ try to deduce current machine values without any
     constraints at all
     """
-    output.writeln("\nIt seems to be the first time you run conan", Color.BRIGHT_YELLOW)
+    output.writeln("\nIt seems to be the first time you've ran conan", Color.BRIGHT_YELLOW)
```

Comment:
Thanks very much for the detailed explanation! I'll go with the first one, it was the natural for you :)